data/migrations/V0.1__create_roles.sql

```diff
@@ -0,0 +1,25 @@
+-- Create the roles needed in the DB if they do not already exist
+-- Unfortunately, Postgres lacks a "CREATE ROLE IF NOT EXISTS ..."
```

Comment:
Oh, also, in keeping with naming conventions going forward and helping with reading the migrations directory, could we rename this file to `V0000.1__create_roles.sql` please?  Some point soon we'll rename the others to have the leading zeros, this way it's easy to scan the directory and see things in order. :-)